robocup_msgs/Edge new_edge
---
bool success
